rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "stable"
+channel = "1.73"
```

Comment:
Why are we pinning to 1.73? Is there some issue with the current stable (1.74)?

Reply:
To prevent future build breakage